app/frontend/private/pos/payment/script.js

```diff
@@ -101,7 +101,8 @@ function handlePaymentButton() {
     const cash = parseFloat(cashInput.value.replace("$", ""));
     const change = cash - total;
 
-    if (cash < total) {
+    if (cash < total.toFixed(2)) {
+        console.log(total);
```

Comment:
don't forget to remove this

(also how did you manage to make total a global variable in updateCalculations() lmao)

Reply:
recommend adding near the top of payment/script.js
```
let discountAmount;
let tipAmount;
let total;
```

Reply:
> don't forget to remove this
> 
> (also how did you manage to make total a global variable in updateCalculations() lmao)

LOL it happens when you don't specify let or const before the variable